tools/pyboard.py

```diff
@@ -125,7 +127,7 @@ def eval(self, expression):
         ret = ret.strip()
         return ret
 
-    def exec(self, command):
+    def exec(self, command, data_consumer=None):
```

Comment:
Hmm. I don't use the data_consumer, so that line could remain the same as it was before.